.mypy.ini

```diff
@@ -11,7 +11,7 @@ warn_unused_ignores = True
 disallow_untyped_defs = True
 
 # 3rd-party libs without type hints nor stubs
-[mypy-matplotlib.*,scipy.*,qutip.*,jsonschema.*]
```

Comment:
bump pasqal-sdk to 0.1.6 and this should not be required

Reply:
Thanks for the remark, I will check it in another PR.

Reply:
I think @awennersteen meant the addition of the `sdk.*` , which I think should be reverted here, no?